### PR TITLE
implement sections 310, 314, 316, 317, 320, 322, 323

### LINKED ISSUE #491: Section 310 Book 1
```
If there is a tick in the box, turn to 614 immediately. If not, read
on.
The Stinking River has cut its way through the high ground
here. On the edge of the chasm that overlooks the river lies the
village of High Therys.
The locals are behaving very oddly. You watch the villagers
lay out a fine feast on the village green. A pig is set roasting on a
spit, loaves of freshly baked bread are piled beside bowls
overflowing with fruits, butter and cheese. Lastly, the richest
merchants of the village bring coffers full of silver which they set
beside the table. To your amazement, instead of sitting down to
dine, the villagers then scurry home and start closing the shutters
on their windows. Turn to 283.
```

**Reference features required (if any)**
TickBox

**Additional context**
none

### LINKED ISSUE #492: Section 314 Book 1
```
The runes mean nothing to you.
Go down to the beach turn to 97
Take the road to Trefoille turn to 602
Take the road to Marlock City turn to 166
```

**Reference features required (if any)**
none

**Additional context**
none

### LINKED ISSUE #493: Section 316 Book 1
```
The temple of Elnir is an imposing edifice of grey stone, inlaid
with yellow marbling. Elnir is the god of the sky and of
kingship. He is the ruler of the gods.
‘His dreams are the clouds before a storm,’ says a passing
priest.
Become an initiate of Elnir turn to 291
Renounce his worship turn to 143
Seek a blessing turn to 388
Leave the temple turn to 10
```

**Reference features required (if any)**
none

**Additional context**
none

### LINKED ISSUE #494: Section 317 Book 1
```
It is the same craft you got the magic spear from, with the silver-skinned demon inside – still dead you presume. You examine it
again, but there is nothing else of interest on board. The figure
still lies there unmoving. You sail on. Turn to 85.

```

**Reference features required (if any)**
none

**Additional context**
none

### LINKED ISSUE #495: Section 320 Book 1
```
You learn from their conversations that they are from the
jungles of distant Ankon-Konu, the Feathered Lands, and that
they have come here to find new sacrifices for their god, and
new members for their cult. The chef pronounces the cauldron
is ready, and they remove the net that holds you.
Fight turn to 204
Shout ‘Badogor!’ repeatedly turn to 101
Join their cult turn to 61
```

**Reference features required (if any)**
none

**Additional context**
none

### LINKED ISSUE #496: Section 322 Book 1
```
The repulsive one recognizes you from your last undersea trip!
Turn to 238.
```

**Reference features required (if any)**
none

**Additional context**
none

### LINKED ISSUE #497: Section 323 Book 1
```
A few locals are discussing recent events – it seems some hideous
beast has made its lair in a cave on the hill that overlooks the
village.
‘It’s a gorlock,’ says a farmer. ‘And it’s amassed quite a bit of
treasure by all accounts – lifted from its victims. A brave
adventurer like yourself could get rich it you killed it. You’d be
doing us a favour as well.’
You ask what a gorlock looks like, but no one knows.
‘Nobody’s come back to tell the tale!’ says the innkeeper.
You leave the tavern. Turn to 510.
```

**Reference features required (if any)**
none

**Additional context**
none

### PR DESCRIPTION
**Pull request by Desi7**
### What?
Implemented Timo's secions as mentioned in title

### Why?
Because needed

### How?
coding

### Testing?
nope

### Anything Else?
closes #491 
closes #492 
closes #493 
closes #494 
closes #495 
closes #496 
closes #497

### CODE REVIEW

**Review by keykey7 — APPROVED** [review]

**Review by Lion-alt — APPROVED** [review]
LGTM 👍

File: fabbwled-backend/src/main/resources/sections/289-324.yaml [hunk 1]
```diff
@@ -15,3 +15,67 @@
...
             - turnTo: 277
+- number: 310
+  content:
+    - text: If there is a tick in the box
+    - if:
+        isTickBoxDone: true
+      then:
+        - turnTo: 614
+      else:
+        - text: If not, read on. The Stinking River has cut its way through the high ground here. On the edge of the chasm that overlooks the river lies the village of High Therys. The locals are behaving very oddly. You watch the villagers lay out a fine feast on the village green. A pig is set roasting on a spit, loaves of freshly baked bread are piled beside bowls overflowing with fruits, butter and cheese. Lastly, the richest merchants of the village bring coffers full of silver which they set beside the table. To your amazement, instead of sitting down to dine, the villagers then scurry home and start closing the shutters on their windows.
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/resources/sections/289-324.yaml:26]: next time, maybe add some line breaks. its fine though

**Review by Nilstrieb — APPROVED** [review]

File: fabbwled-backend/src/main/resources/sections/289-324.yaml (hunk 1 above)
Comment (Desi7) [reply fabbwled-backend/src/main/resources/sections/289-324.yaml:26]: 👍

**Review by kaibria — APPROVED** [review]
lgtm

**Review by tamer545 — APPROVED** [review]

**Pull request merged**